tools/protocol-schema-check/res/protocol_schema.toml

```diff
@@ -3,11 +3,11 @@ AccessKeyPermission = 885623561
 Account = 358811118
 AccountV2 = 337859929
 AccountVersion = 4249996519
-Action = 2541352025
+Action = 1072020253
```

Comment:
Wow, that's a LOT of things changing due to a simple change in Action!

Reply:
yeah, I was also surprised by that!